bxin/ptg2m2/ptg2m2.ipynb

```diff
@@ -232,7 +232,7 @@
     "cmd_radec = cmd_elaz.transform_to(ICRS)\n",
     "# Calculating the other parameters     \n",
     "rot_pa = rot_tel\n",
-    "await rot.cmd_trackStart.start(timeout=30.)\n"
+    "#await rot.cmd_trackStart.start(timeout=30.)"
```

Comment:
Please delete this line. Leaving commented-out code is generally bad form, as it can lead to bit rot. But in this case it's much worse because if anyone uncomments the line it will break the script.

Reply:
ah, I fixed it in the hexapod notebook, and was going to do that for the m2 notebook later. Fixed now...